### PR TITLE
chore: remove component getters

### PR DESCRIPTION
**Pull request by kvet**

### CODE REVIEW

File: packages/dx-react-grid/docs/reference/table-edit-column.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Name | Type | Default | Description
 -----|------|---------|------------
 cellComponent | ElementType&lt;[TableEditColumnCellProps](#tableeditcolumncellprops)&gt; | | A component that renders a command cell within a data row.
 headerCellComponent | ElementType&lt;[TableEditColumnHeaderCellProps](#tableeditcolumnheadercellprops)&gt; | | A component that renders a command cell within the header row.
-getCommandComponent | (id: 'add' &#124; 'edit' &#124; 'delete' &#124; 'commit' &#124; 'cancel') => ElementType&lt;[EditCommandProps](#editcommandprops)&gt; | | A function returning a component that renders command controls a specific command.
+commandComponent | ElementType&lt;[EditCommandProps](#editcommandprops)&gt; | | A component that renders command controls.
```
Comment (gsobolev) [inline packages/dx-react-grid/docs/reference/table-edit-column.md]: It's not clear to me whether this component renders just one control or multiple ones.

File: packages/dx-react-grid/src/plugins/table-edit-column.test.jsx [hunk 2]
```diff
@@ -142,10 +135,10 @@ describe('TableHeaderRow', () => {
         </PluginHost>
       ));
 
-      expect(tree.find(defaultCommandComponents.add).props())
+      expect(tree.find(defaultProps.commandComponent).filterWhere(wrapper => wrapper.prop('id') === 'add').props())
```
Comment (gsobolev) [inline packages/dx-react-grid/src/plugins/table-edit-column.test.jsx]: Try to refactor the similar lines somehow (e.g. extract helper etc.).

File: packages/dx-react-demos/src/bootstrap3/featured-controlled-mode/demo.jsx [hunk 3]
```diff
@@ -62,103 +56,103 @@ CommandButton.defaultProps = {
...
 };
 
+const Cell = (props) => {
+  if (props.column.name === 'discount') {
+    return <ProgressBarCell {...props} />;
+  }
+  if (props.column.name === 'amount') {
+    return <HighlightedCell {...props} />;
+  }
+  return <Table.Cell {...props} />;
```
Comment (SergeyAlexeev) [inline packages/dx-react-demos/src/bootstrap3/featured-controlled-mode/demo.jsx]: Can we return `undefined` here?

Comment (kvet) [reply packages/dx-react-demos/src/bootstrap3/featured-controlled-mode/demo.jsx]: No, a React component can't return `undefined`.

File: packages/dx-react-grid/src/plugins/table-edit-column.test.jsx [hunk 4]
```diff
@@ -175,13 +168,13 @@ describe('TableHeaderRow', () => {
           ...defaultDeps.template.tableCell,
           row: defaultDeps.template.tableCell.tableRow.row,
         });
-      expect(tree.find(defaultCommandComponents.edit).exists())
+      expect(tree.find(defaultProps.commandComponent).filterWhere(wrapper => wrapper.prop('id') === 'edit').exists())
```
Comment (SergeyAlexeev) [inline packages/dx-react-grid/src/plugins/table-edit-column.test.jsx]: Too long string `defaultProps.commandComponent).filterWhere(wrapper => wrapper.prop('id') `. Can we extract it?

**Pull request merged**